### PR TITLE
Add alertmanager HA support section 

### PR DESCRIPTION
**Pull request by josedonizetti**
Add brief section about Alermanager HA support. https://github.com/prometheus/docs/issues/844

@brian-brazil

### CODE REVIEW

File: content/docs/alerting/alertmanager.md [hunk 1]
```diff
@@ -67,3 +67,8 @@ Silences are configured in the web interface of the Alertmanager.
 The Alertmanager has [special requirements](../clients) for behavior of its
 client. Those are only relevant for advanced use cases where Prometheus
 is not used to send alerts.
+
+## High Availability
+
+Alertmanager supports a mesh configuration to create a cluster for high availability.
```
Comment (brian-brazil) [inline content/docs/alerting/alertmanager.md]: I'd also mention here that all Prometheus servers talk to all Alertmanagers, as that's a common misunderstanding.

File: content/docs/alerting/alertmanager.md [hunk 2]
```diff
@@ -67,3 +67,14 @@ Silences are configured in the web interface of the Alertmanager.
...
+This can be configured using the [-mesh-*](https://github.com/prometheus/alertmanager#high-availability) flags.
+
+It's important not to load balance traffic between Prometheus and its Alertmanagers, but instead, point Prometheus to a list of all Alertmanagers.
+
+The Alertmanager implementation expects all alerts to be sent to all Alertmanagers to ensure high availability.
```
Comment (brian-brazil) [inline content/docs/alerting/alertmanager.md]: This repeats the above information.

File: content/docs/alerting/alertmanager.md [hunk 3]
```diff
@@ -67,3 +67,14 @@ Silences are configured in the web interface of the Alertmanager.
...
+You can use the `-alertmanager.url` when starting
```
Comment (brian-brazil) [inline content/docs/alerting/alertmanager.md]: This is outdated with Prometheus 2.0 coming along which removed this flag. This also belongs with docs on how to hook the two together, not just in the HA section. I'd remove this.

**Comment by josedonizetti** [comment]
@brian-brazil Let me know if anything else needs change. Thanks.

File: content/docs/alerting/alertmanager.md (hunk 1 above)
Comment (brancz) [reply content/docs/alerting/alertmanager.md]: I do think this is important, could you add this @josedonizetti ?

Comment (josedonizetti) [reply content/docs/alerting/alertmanager.md]: @brancz It's already added. Paragraph below.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**